src/actions/measurements.js

```diff
@@ -263,34 +264,45 @@ export const parseMeasurementsJSON = (json) => {
     );
   });
 
-  return {collections, defaultCollection: json["default_collection"]};
-};
-
-export const loadMeasurements = ({collections, defaultCollection}) => (dispatch, getState) => {
-  const { tree, controls } = getState();
-  if (!tree.loaded) {
-    throw new Error("tree not loaded");
-  }
-
   const collectionKeys = collections.map((collection) => collection.key);
-  let defaultCollectionKey = defaultCollection;
+  let defaultCollectionKey = json["default_collection"];
   if (!collectionKeys.includes(defaultCollectionKey)) {
     defaultCollectionKey = collectionKeys[0];
   }
-
-  // Get the collection to display to set up default controls
-  const collectionToDisplay = getCollectionToDisplay(collections, controls.measurementsCollectionKey, defaultCollectionKey);
-  const newControls = getCollectionDisplayControls(controls, collectionToDisplay);
-  const queryParams = createMeasurementsQueryFromControls(newControls, collectionToDisplay, defaultCollectionKey);
-
-  dispatch({
-    type: LOAD_MEASUREMENTS,
+  const collectionToDisplay = collections.filter((collection) => collection.key === defaultCollectionKey)[0];
+  return {
+    loaded: true,
+    error: undefined,
     defaultCollectionKey,
     collections,
-    collectionToDisplay,
-    controls: newControls,
-    queryParams
-  });
+    collectionToDisplay
+  }
+};
+
+export const loadMeasurements = (measurementsData, dispatch) => {
+  let measurementState = getDefaultMeasurementsState();
+  let warningMessage = "";
+  if (measurementsData === undefined) {
+    // eslint-disable-next-line no-console
+    console.debug("No measurements JSON fetched");
```

Comment:
[aside]

We can strip these out of production builds if you want?

Reply:
Removed the console message as part of https://github.com/nextstrain/auspice/pull/1881/commits/0069e09eb66a8a3654b660d881bcb39dac765cd1